CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 
 * [#1216](https://github.com/ruby-grape/grape/pull/1142): Fix JSON error response when calling `error!` with non-Strings - [@jrforrest](https://github.com/jrforrest).
 * [#1225](https://github.com/ruby-grape/grape/pull/1225): Fix `given` with nested params not returning correct declared params - [@JanStevens](https://github.com/JanStevens).
+* [#1249](https://github.com/ruby-grape/grape/pull/1249): Don't fail even if invalid type value is provided - [@namusyaka](https://github.com/namusyaka).
 
```

Comment:
We should specify the scope, it's only related to the params with `default` validator :bowtie: